lib/output.js

```diff
@@ -58,7 +58,7 @@ function OutputStream(options) {
         beautify         : false,
         bracketize       : false,
         comments         : false,
-        ecma             : 5,
+        ecma             : 8,
```

Comment:
I think the output `ecma` default should remain `5`. 
See rationale: https://github.com/mishoo/UglifyJS2/pull/2157#discussion_r123872481